### ISSUE TITLE
Failed to parse URL from tree-sitter.wasm on VSCode 1.82

### ISSUE THREAD

**Issue by CyprusSocialite** [cucumber/vscode#174]
### 👓 What did you see?

On start-up [Extension Host (Remote)] shows the following error:
```
2023-09-11 16:07:13.194 [error] TypeError: Failed to parse URL from ~/.vscode-server/extensions/cucumberopen.cucumber-official-1.7.0/out/tree-sitter.wasm
	at Object.fetch (node:internal/deps/undici/undici:11413:11)
```

### 📦 Which tool/library version are you using?

Extension v1.7.0

VSCode 1.82.0

WSL 2

----
 
*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/ISSUE_TEMPLATE/bug_report.md)*

**Comment by CyprusSocialite** [comment]
Does not seem to happen in VSCode 1.81 by the way

**Comment by lmexo** [comment]
problem persists on code 1.84

**Comment by mkmkmk** [comment]
I have the same problem. I downgraded to 1.81 and now it's OK.

**Comment by kieran-ryan** [comment]
@CyprusSocialite or @mkmkmk, does [manually installing](https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix) the [v1.8 release](https://github.com/cucumber/vscode/releases/tag/release%2Fv1.8.0) (.vsix) resolve the issue for you? There was a tree-sitter fix in that release (see #164) that will be available in the marketplace once we resolve a publishing issue. Wondering is this a duplicate of that issue. And whether relates to #121?

**Comment by kieran-ryan** [comment]
Considered closed through v1.8.0 release. Issue will be reopened if further information or a Minimum Reproducible Example provided requiring a fix.
